layouts/partials/news.html

```diff
@@ -0,0 +1,12 @@
+<div class="container rows">
+    {{ range . }}
```

Comment:
This `.` is what we are passing into the partial. In the example below it is the blog posts data limited by 4.

```
{{ $posts := (where .Site.RegularPages "Section" "blog" | first 4) }}
{{ partial "news.html" $posts }}
```

Reply:
ahh.